recipes/rocksdb/all/conanfile.py

```diff
@@ -0,0 +1,131 @@
+from conans import ConanFile, CMake, tools
+from conans.tools import Version
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class RocksDB(ConanFile):
+    name = "rocksdb"
+    homepage = "https://github.com/facebook/rocksdb"
+    license = "GPL-2.0"
```

Comment:
```suggestion
    license = ("GPL-2.0-only", "Apache-2.0")
```

https://github.com/facebook/rocksdb#license